system/include/emscripten/wasmfs.h

```diff
@@ -69,6 +69,15 @@ backend_t wasmfs_create_opfs_backend(void);
 backend_t wasmfs_create_icase_backend(backend_constructor_t create_backend,
                                       void* arg);
 
+// Similar to fflush(0), but also flushes all internal buffers inside WasmFS.
+// This is necessary because in a Web environment we must buffer at an
+// additional level after libc, since console.log() prints entire lines, that
+// is, we can't print individual characters as libc feeds them to us, so we
+// buffer them and call console.log() only after a newline. This function will
+// actually flush all buffers and add newlines as necessary to get everything
+// printed out.
+void wasmfs_flush(void);
```

Comment:
Maybe this should be called `wasmfs_flush_streams` or similar, since it only applies to the standard streams. Unless the intention is that in the future this will flush everything in the entire file system?

Reply:
Yes, exactly, there is a TODO for that in the code. I think it would be good to just flush everything from this call.